clicommand/pipeline_upload.go

```diff
@@ -301,6 +316,24 @@ var PipelineUploadCommand = cli.Command{
 			}
 		}
 
+		if cfg.ApplyIfChanged {
+			func() {
```

Comment:
should this be lifted out, rather than inline?

Reply:
Have done that now.